python/cugraph/traversal/bfs.py

```diff
@@ -26,10 +26,6 @@ def bfs(G, start, directed=True):
         as an adjacency list.
     start : Integer
         The index of the graph vertex from which the traversal begins
-    directed : bool
```

Comment:
Only the doc was removed but the API still has the option

Reply:
ah, looks like i forgot to revert that when I moved the bfs fix to a different PR.